src/App.css

```diff
@@ -22,12 +24,32 @@
     justify-content: space-around;
 }
 
-.App-tabs > a {
+.App-tabs > div {
     margin-left: 20px;
     margin-right: 20px;
     cursor: pointer;
 }
 
 .bold {
     font-weight: bold;
+}
+
+.flex {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.center-item {
+    justify-content: center;
+    align-content: center;
+}
+
+.App-container {
+    height: 100vh;
+    display: flex;
+}
+
+.App-title {
+    font-size: 26px;
```

Comment:
Minor: maybe we should start using em for font sizing so it scales better with different screen sizes.

Reply:
that's a good call